examples/cloudngfw_distributed/example.tfvars

```diff
@@ -0,0 +1,526 @@
+  Owner       = "PS Team"
+}
+
+ssh_key_name = "example-ssh-key" # TODO: update here
+
+### VPC
+vpcs = {
+  app1_vpc = {
+    name  = "app1-spoke-vpc"
+    cidr  = "10.104.0.0/16"
+    nacls = {}
+    security_groups = {
+      app1_vm = {
+        name = "app1_vm"
+        rules = {
+          all_outbound = {
+            description = "Permit All traffic outbound"
+            type        = "egress", from_port = "0", to_port = "0", protocol = "-1"
+            cidr_blocks = ["0.0.0.0/0"]
+          }
+          ssh = {
+            description = "Permit SSH"
+            type        = "ingress", from_port = "22", to_port = "22", protocol = "tcp"
+            cidr_blocks = ["1.1.1.1/32", "10.104.0.0/16", "10.105.0.0/16"] # TODO: update here (replace 1.1.1.1/32 with your IP range)
+          }
+          https = {
+            description = "Permit HTTPS"
+            type        = "ingress", from_port = "443", to_port = "443", protocol = "tcp"
+            cidr_blocks = ["1.1.1.1/32", "10.104.0.0/16", "10.105.0.0/16"] # TODO: update here (replace 1.1.1.1/32 with your IP range)
+          }
+          http = {
+            description = "Permit HTTP"
+            type        = "ingress", from_port = "80", to_port = "80", protocol = "tcp"
+            cidr_blocks = ["1.1.1.1/32", "10.104.0.0/16", "10.105.0.0/16"] # TODO: update here (replace 1.1.1.1/32 with your IP range)
+          }
+        }
+      }
+      app1_lb = {
+        name = "app1_lb"
+        rules = {
+          all_outbound = {
+            description = "Permit All traffic outbound"
+            type        = "egress", from_port = "0", to_port = "0", protocol = "-1"
+            cidr_blocks = ["0.0.0.0/0"]
+          }
+          https = {
+            description = "Permit HTTPS"
+            type        = "ingress", from_port = "443", to_port = "443", protocol = "tcp"
+            cidr_blocks = ["1.1.1.1/32"] # TODO: update here (replace 1.1.1.1/32 with your IP range)
+          }
+          http = {
+            description = "Permit HTTP"
+            type        = "ingress", from_port = "80", to_port = "80", protocol = "tcp"
+            cidr_blocks = ["1.1.1.1/32"] # TODO: update here (replace 1.1.1.1/32 with your IP range)
+          }
+        }
+      }
+    }
+    subnets = {
+      # Do not modify value of `set=`, it is an internal identifier referenced by main.tf.
+      "10.104.0.0/24"   = { az = "eu-west-1a", set = "app1_vm", nacl = null }
+      "10.104.128.0/24" = { az = "eu-west-1b", set = "app1_vm", nacl = null }
+      "10.104.2.0/24"   = { az = "eu-west-1a", set = "app1_lb", nacl = null }
+      "10.104.130.0/24" = { az = "eu-west-1b", set = "app1_lb", nacl = null }
+      "10.104.3.0/24"   = { az = "eu-west-1a", set = "cngfw_subnet", nacl = null }
+      "10.104.131.0/24" = { az = "eu-west-1b", set = "cngfw_subnet", nacl = null }
+      "10.104.4.0/24"   = { az = "eu-west-1a", set = "natgw", nacl = null }
```

Comment:
The natgw subnet should be located in security VPC - we dont wont to promote separate NATGW because it requires to many endpoints and cngfw can reach quickly limit of 300 endpoints in distributed architecture